src/actions/types.js

```diff
@@ -25,7 +26,12 @@ export type Action = | {type: 'AUTH_MYSTERY'}
  | {type: 'LOGIN_FAILURE', errors: any}
  | {type: 'RECEIVE_COFFEE', payload: Coffee, meta: {id: string}}
  | {type: 'UPDATE_COFFEE', payload: Coffee, meta: {id: string}}
- | {type: 'DELETE_COFFEE', meta: {id: string}};
+ | {type: 'DELETE_COFFEE', meta: {id: string}}
+ | {type: 'RECEIVE_BREW', payload: Brew, meta: {id:string}}
+ | {type: 'UPDATE_BREW', payload: Brew, meta: {id: string}}
+ | {type: 'DELETE_BREW', meta: {id: string}};
 
 // anyhow, for now we don't have any actions (yet!) but we will!! and that is how we
 // will type them for our reducers
+
+// soooooooo the above arent actually actions? -cb
```

Comment:
that comment is stale! i was just making action types without having the actions yet xD